README.md

```diff
@@ -23,9 +23,8 @@ Detect security vulnerabilities and compliance violations across your Infrastruc
 
 ## Quick Start 
 ### Step 1: Install
-Terrascan's binary for your architecture can be found on the [releases](https://github.com/accurics/terrascan/releases)
-
-Terrascan's binary for your architecture can be found on the [releases](https://github.com/accurics/terrascan/releases) page. Here's an example of how to install it:
+Terrascan's supports multiple ways to install, including brew (see more below).
```

Comment:
we can add a hypelink on "including brew" to point at https://github.com/accurics/terrascan#install-via-brew .. will save us from "(see more below)".